CHANGELOG.md

```diff
@@ -6,6 +6,22 @@
 
 * More accurately returns 503 instead of 500 when Cromwell can not respond in a timely manner
 
+### GPU
+
+The PAPI backend now supports specifying GPU through WDL runtime attributes:
+
+```wdl
+runtime {
+    gpuType: "nvidia-tesla-k80"
+    gpu: 2
```

Comment:
@cjllanwarne You might want to weigh in on this syntax
I have it as `gpu` instead of `gpuCount` or something else to mirror the existing `cpu` but we could want something different

Reply:
Now that you mention it I think `gpuCount` is more descriptive

Reply:
The only alternative would be a map/object style, something like
```
gpu: object {
  type: "nvidia-tesla-k80",
  count: 2
}
```

Reply:
If that `object` could be sugared out of existence that would be even nicer syntax.